hack/all-in-one/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fedora:33
```

Comment:
Should this be the rhel8 ubi rather than fedora?

Reply:
I cannot find instructions how to install crio on ubi